### PR TITLE
add cassandra cli

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: repo/packages/C/cassandra/1/command.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "pip": [
+    "dcos<1.0",
```
Comment (elingg) [inline repo/packages/C/cassandra/1/command.json]: What is "dcos<1.0"?

File: repo/packages/C/cassandra/1/command.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "git+https://github.com/mesosphere/dcos-cassandra.git#dcos-cassandra=0.1.0"
+  ]
```
Comment (elingg) [inline repo/packages/C/cassandra/1/command.json]: Ideally we'll use whl for this, see https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/S/spark/1/command.json with an automated build, but this also works

**Comment by elingg** [comment]
Thanks @tamarrow! Was this tested?

File: repo/packages/C/cassandra/1/command.json (hunk 2 above)
Comment (kensipe) [inline repo/packages/C/cassandra/1/command.json]: +1 on the wheel

**Comment by kensipe** [comment]
@tamarrow @elingg I just tested this... I was going to sneak it in prior to asking for a PR on multiverse.. however it fails:

```
dcos cassandra
Traceback (most recent call last):
  File "/Users/kensipe/.dcos/subcommands/cassandra/env/bin/dcos-cassandra", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/Users/kensipe/.dcos/subcommands/cassandra/env/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3084, in <module>
    @_call_aside
  File "/Users/kensipe/.dcos/subcommands/cassandra/env/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3070, in _call_aside
    f(*args, **kwargs)
  File "/Users/kensipe/.dcos/subcommands/cassandra/env/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3097, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/Users/kensipe/.dcos/subcommands/cassandra/env/lib/python2.7/site-packages/pkg_resources/__init__.py", line 653, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/Users/kensipe/.dcos/subcommands/cassandra/env/lib/python2.7/site-packages/pkg_resources/__init__.py", line 666, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/Users/kensipe/.dcos/subcommands/cassandra/env/lib/python2.7/site-packages/pkg_resources/__init__.py", line 839, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'dcos==0.1.13' distribution was not found and is required by dcoscassandra
```

**Comment by kensipe** [comment]
@tamarrow any help here?

File: repo/packages/C/cassandra/1/command.json (hunk 2 above)
Comment (tamarrow) [inline repo/packages/C/cassandra/1/command.json]: the automated build -> whl isn't set up yet, so lets use it as is for now and switch when we updated the dcos-cassandra repo?

File: repo/packages/C/cassandra/1/command.json (hunk 1 above)
Comment (tamarrow) [inline repo/packages/C/cassandra/1/command.json]: I changed this to dcos==0.1.13 which is the version specified in dcos-cassandra cli. This fixes @kensipe's issue.

**Comment by tamarrow** [comment]
@elingg i tested this locally using the cli

**Comment by kensipe** [comment]
looking...

**Comment by kensipe** [comment]
LGTM... merging!

**Pull request merged**